source/draw/Canvas.ooc

```diff
@@ -21,19 +21,13 @@ InterpolationMode: enum {
 
 Canvas: abstract class {
 	_size: IntVector2D
-	_transform := FloatTransform3D identity
 	size ::= this _size
 	viewport: IntBox2D { get set }
-	blend: Bool { get set }
-	opacity: Float { get set }
-	transform: FloatTransform3D { get { this _transform } set(value) { this _transform = value } }
```

Comment:
The transform was not even used by the legacy draw calls on the CPU. Just blit operations for resizing things.